docs/blog/posts/argocd-keptn-health.md

```diff
@@ -0,0 +1,180 @@
+and configured on your cluster, which is able to fetch the `response time` values
+of the microservices.
+In our setup, we are going to use [Prometheus](https://prometheus.io/).
+
+First, we need to create [KeptnMetric](https://keptn.sh/stable/docs/reference/crd-reference/metric/)
+and [KeptnMetricsProvider](https://keptn.sh/stable/docs/reference/crd-reference/metricsprovider/)
+resources in our cluster.
+These two resources contain a simple query for fetching `response time` of the `podtato-head`
+application microservice as well as configuration for the metrics provider supplying the data.
+
+```yaml
+{% include "./argocd-keptn-health/metric.yaml" %}
+```
+
+Next, we add `KeptnEvaluationDefinition` into our git repository, where our
+`podtato-head` application lives.
+It defines the [SLO](https://www.dynatrace.com/news/blog/what-are-slos/)
+by linking the existing `KeptnMetric` resource and providing the rule the value should fullfil.
+
+```yaml
+{% include "./argocd-keptn-health/evaluationdefinition.yaml" %}
+```
+
+Additionally, we annotate the `podtato-head-frontend` Deployment to execute
+the task as part of `post-deployment-evaluation` checks.
+
+```yaml
+{% include "./argocd-keptn-health/annotation.yaml" %}
+```
+
+After these two changes are made in our git repository, ArgoCD will see changes and re-trigger
+the deployment of `podtato-head`.
+Keptn waits until all of the
+application pods are running and afterwards, it executes `post-deployment-evaluation` tasks.
+
+Due to slow `response time` of the `podtato-head-frontend` microservice, the
+executed `KeptnEvaluation` fails.
+
+Here we can see that with the use of Keptn we can perform more advanced health checks
+(tasks or evaluations) and verify that the application is healthy during the process
+of deployment which is performed by ArgoCD.
+Similarly to `KeptnEvaluations`, `KeptnTasks` can be executed as part of a quality health check
+for our application.
+
+## How to show Keptn health status in ArgoCD UI?
+
+Using Keptn together with ArgoCD brings a lot of value, which we saw in the previous section,
+but observing application health status by inspecting the status of the
+various resources using `kubectl` is not the best user experience.
+The data should be nicely displayed in the ArgoCD UI to provide the user with an overview
+if the application was deployed, if it's synchronized and if it's healthy, all in
+one place.
+
+Due to this reason, we are going to implement an ArgoCD UI extension with additional application health
+data, which are retrieved from Keptn.
+This way, the ArgoCD UI will act as a single source of truth for the user providing all
+the information about the deployed application.
+
+Below you can see the first mockups how the extension of ArgoCD UI might look like
+and how a failed `KeptnEvaluation` and therefore unhealthy Keptn status of `podtato-head-frontend`
+Deployment might be displayed on the main ArgoCD UI screen.
```

Comment:
```suggestion
deployment might be displayed on the main ArgoCD UI screen.
```

Reply:
Here I am trying to refer to a k8s Deployment, I would rather keep it capitalized